spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/r2dbc/R2dbcProperties.java

```diff
@@ -153,6 +171,11 @@ public String determineUniqueName() {
 		 */
 		private String validationQuery;
 
+		/**
+		 * Validation depth.
+		 */
+		private ValidationDepth validationDepth = ValidationDepth.LOCAL;
```

Comment:
Enum default value are not detected by the annotation processor so additional metadata has to be written for that unfortunately.

Here is [an example](https://github.com/spring-projects/spring-boot/blob/64aac04862cd129dea2f8e929fb977f1138806cb/spring-boot-project/spring-boot-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json#L498-L501).